### DISCUSSION TITLE
Catppuccin in options

### DISCUSSION THREAD

**Discussion by uldrim** [LazyVim/LazyVim#1033] [Q&A] [answered]
I am using ubuntu 22.04 + wezterm + neovim 0.9.1 (built from source)
I see that catppuccin is installed by default (while updating LazyVim) but I can't see it when i press <Space> + <u> + <C>.
Is there any way to get all flavours of catppuccin available there along with tokyonight variants.
Thank you for your response.

**Comment by dpetka2001** [comment]
There's a keybinding `<leader>uC` that shows the colorschemes available in a Telescope picker with preview enabled.

**Reply by dpetka2001** [reply, marked as answer]
I think you can't see it cuz it's lazy loaded, since it has `lazy = true`. So it will only be loaded when you do `:colorscheme catppuccin`. I have changed this in my personal spec to also include the following
```lua
    keys = {
      { "<leader>uC", Util.telescope("colorscheme", { enable_preview = true }), desc = "Colorscheme with preview" },
    },
```
so that it will also be loaded in the Telescope picker with preview available.

**Reply by dpetka2001** [reply]
Please see my previous post.

**Reply by uldrim** [reply]
Ohh, that might be it.
Thank you so much.

**Reply by dpetka2001** [reply]
The whole config would be like 
```lua
  {
    "catppuccin/nvim",
    keys = {
      { "<leader>uC", Util.telescope("colorscheme", { enable_preview = true }), desc = "Colorscheme with preview" },
    },
  },
```
If you put it in a file in `lua/plugins` directory (I put mine in `lua/plugins/colorscheme.lua`), you should be able to also show it with the already mentioned keybinding.
